conf/modules/guidance_hybrid.xml

```diff
@@ -12,16 +12,16 @@
         <dl_setting var="max_airspeed" MIN="4" STEP="1" MAX="160" module="guidance/guidance_hybrid"/>
         <dl_setting var="wind_gain" MIN="1" STEP="1" MAX="256"/>
         <dl_setting var="horizontal_speed_gain" MIN="1" STEP="1" MAX="15" shortname="hor_speed_div"/>
-        <dl_setting var="max_turn_bank" MIN="10.0" STEP="1.0" MAX="60.0" shortname="max_turn_bank"/>
-        <dl_setting var="turn_bank_gain" MIN="0.1" STEP="0.01" MAX="3.0" shortname="turn_bank_gain"/>
+        <dl_setting var="max_turn_bank" MIN="10.0" STEP="1.0" MAX="60.0"/>
+        <dl_setting var="turn_bank_gain" MIN="0.1" STEP="0.01" MAX="3.0"/>
 
-        <dl_setting var="cruise_throttle" MIN="0" STEP="1" MAX="9600" shortname="cruis_throttle"/>
```

Comment:
Cruise Throttle

Reply:
I have removed the `shortname` instead of correcting the spelling since it was the same than `name` (so it makes no difference in the end)